### ISSUE TITLE
add tags to org view bug

### ISSUE THREAD

**Issue by JMurphyWeb** [emfoundation/ce100-app#431]
endpoint: orgs/1/tags 

Clicking on a tag does not always select it. It could be to do with the id's and no longer using @index.

Look into it

**Comment by JMurphyWeb** [comment]
Specifically, logging in with sa@ro.co and adding tags to Apple. if you try to deselect MemberType//Global Partner it just closes the Member Type category.

**Comment by JMurphyWeb** [comment]
Fixed by prefixing the category id with `cat_`:
```
<label class="tag__title" for="cat_{{category_id}}">{{category_name}}</label>
```

**Comment by SimonLab** [comment]
resolved with #432
